pypesto/sampling/sampler.py

```diff
@@ -32,7 +32,7 @@ def initialize(self,
 
     @abc.abstractmethod
     def sample(
-            self, n_samples: int, beta: float = 1.
+            self, n_samples: int, beta: float = 1., temper_lpost: bool = False
```

Comment:
Code format: No line break before `self`. One line per argument...

Reply:
it depends -- sometimes we do one argument per line, but if it fits in one or two lines all-together, we put all bunched. always trying to make it as readable as possible -- is it more important to have a concise code (and not have to look up 100 lines of arguments) or can't we find an argument if it's all just lined up.

Reply:
so this is fine as it is?